packages/react-native-bridge/android/build.gradle

```diff
@@ -1,8 +1,11 @@
 buildscript {
-    def buildGutenbergMobileJSBundle =
-            System.getenv('SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD').asBoolean()
-            ? false : (rootProject.ext.has("buildGutenbergMobileJSBundle")
-                    && rootProject.ext.buildGutenbergMobileJSBundle)
+    // def buildGutenbergMobileJSBundle =
+    //         System.getenv('SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD').asBoolean()
+    //         ? false : (rootProject.ext.has("buildGutenbergMobileJSBundle")
+    //                 && rootProject.ext.buildGutenbergMobileJSBundle)
```

Comment:
We'll need to uncomment these lines (and the duplicated lines around line 50) to get the WPAndroid build-from-source build working.